lib/note-editor/index.tsx

```diff
@@ -68,7 +68,7 @@ export class NoteEditor extends Component {
       const prevEditorMode = this.props.editorMode;
       const nextEditorMode = prevEditorMode === 'edit' ? 'markdown' : 'edit';
 
-      this.props.setEditorMode({ mode: nextEditorMode });
+      this.props.setEditorMode(nextEditorMode);
```

Comment:
referring to my comment from below this place also appears to be asking for a `TOGGLE_EDITOR_MODE` action and were to add that we could remove the duplicated logic for setting the `nextEditorMode`

```ts
const mapDispatchToProps: S.MapDispatch<DispatchProps> = {
  closeNote,
  toggleEditorMode,
}

handleShortcut = event => {
  …

  if ( … ) {
    this.props.toggleEditorMode();
    event.stopPropagation();
    event.preventDefault();
    return false;
  }
}
```